### ISSUE TITLE
enum_def could derive IdenStatic

### ISSUE THREAD

**Issue by jclulow** [SeaQL/sea-query#721]
Looking at the **enum_def** attribute macro, it seems like it only produces an **Iden** implementation, but I'm not sure there is any reason it couldn't _also_ derive an **IdenStatic** implementation?

I'd like to use the **IdenStatic.as_str()** routine for column variants in **rusqlite** row conversion routines.  The **Row.get()** routine will accept a **&str** as an index, which would then avoid allocating for column names on each row conversion.

Does that seem reasonable?

**Comment by tyt2y3** [comment]
Yes, you are right. `IdenStatic` was added much later than the `enum_def` macro, that's why. PR is welcome.
